Doctrine/Orm/DataProvider.php

```diff
@@ -107,7 +107,9 @@ public function getCollection($page, array $filters, $itemsPerPage = 30, $order
         }
 
         if ($order) {
-            $queryBuilder->addOrderBy('o.id', $order);
+            foreach ($order as $propName => $value) {
```

Comment:
If order is now an array, you have to update the `getCollection` prototype and change the name of the argument.
@dunglas note that it'll have to be reflected on interface `DataProviderInterface` and it'll already break BC !